### DISCUSSION TITLE
Git flow nutzen?

### DISCUSSION THREAD

**Discussion by janwalenda** [copybarra/chat-client#15] [Ideas]
Git flow ist ein fantastisches Tool um die Pullrequests/Branches um vereinheitlichen und liefert ein Modell dies auch bestens zu nutzen. Nun sollten wir evaluieren ob es sich für uns lohnen würde.

**Comment by dvdkwei** [comment]
haha mag den Title

**Reply by janwalenda** [reply]
huch

**Reply by janwalenda** [reply]
hab das vom handy aus geschrieben
